core/network/impl/CMakeLists.txt

```diff
@@ -29,3 +38,29 @@ add_library(extrinsic_observer
 target_link_libraries(extrinsic_observer
     logger
     )
+
+add_library(remote_sync_protocol_client
+    remote_sync_protocol_client.cpp
+    remote_sync_protocol_client.hpp
```

Comment:
here and below: not necessarily to add hpps. you can safely remove them

Reply:
answered above